src/components/pages/datasets/pep/PepDetail.jsx

```diff
@@ -0,0 +1,248 @@
+  };
+  const [openCriminalProceedings, setOpenCriminalProceedings] = useState(false);
+  const toggleOpenCriminalProceedings = () => {
+    setOpenCriminalProceedings(!openCriminalProceedings);
+  };
+  return (isDataReady ? (
+    <>
+      <div className="intro-y flex items-center mt-8">
+        <h2 className="text-lg mr-auto">
+          {t('pepInfo')}
+        </h2>
+        {/* TODO: Add 'Return to search results' button */}
+        {/*{history.location.state?.fromProjects && (*/}
+        {/*  <Button onClick={() => history.goBack()}
+          className="bg-opacity-0 text-blue-800 h-2 mt-3">*/}
+        {/*    <ArrowLeft className="w-10 h-5" />*/}
+        {/*    <span className="underline">{t('returnToTheProject')}</span>*/}
+        {/*  </Button>*/}
+        {/*)}*/}
+      </div>
+      <div className="col-span-12 lg:col-span-6">
+        <div className="intro-y space-y-1 mt-8 box">
+          <div className="py-4 pl-5 border-b border-gray-200">
+            <h2 className="text-2xl font-medium mr-auto capitalize">
+              {i18n.language === 'uk' ? data.fullname : data.fullname_en}
+            </h2>
+          </div>
+          <div className="pl-5 flex flex-row">
+            <div className="w-64 font-medium">ID:</div>
+            <div className="max-w-xl">{data.id}</div>
+          </div>
+          <div className="pl-5 flex flex-row">
+            <div className="w-64 font-medium">{t('updatedAt')}:</div>
+            <div className="max-w-xl">{dateFormatISO(data.updated_at)}</div>
+          </div>
+          <div className="pl-5 flex flex-row">
+            <div className="w-64 font-medium">{t('status')}:</div>
+            <div className="max-w-xl">{data.is_pep ? t('politicallyExposedPerson') : t('notPoliticallyExposedPerson')}</div>
+          </div>
+          <div className="pl-5 flex flex-row">
+            <div className="w-64 font-medium">{t('pepDetailType')}:</div>
+            <div className="max-w-xl">{data.pep_type_display}</div>
+          </div>
+          <div className="pl-5 flex flex-row">
+            <div className="w-64 font-medium">{t('terminationDatePep')}:</div>
+            <div className="max-w-xl">{data.termination_date ? dateFormatISO(data.termination_date) : '---'}</div>
+          </div>
+          <div className="pl-5 flex flex-row">
+            <div className="w-64 font-medium">{t('reasonOfTermination')}:</div>
+            <div className="max-w-xl">{data.reason_of_termination || '---'}</div>
+          </div>
+          <div className="pl-5 flex flex-row">
+            <div className="w-64 font-medium">{t('dateOfBirth')}:</div>
+            <div className="max-w-xl">{data.date_of_birth || '---'}</div>
+          </div>
+          <div className="pl-5 flex flex-row">
+            <div className="w-64 font-medium">{t('placeOfBirth')}:</div>
+            <div className="max-w-xl">{i18n.language === 'uk' ? data.place_of_birth || '---' : data.place_of_birth_en || '---'}
```

Comment:
create function for get field on uk or en locale

Reply:
done